README.md

```diff
@@ -96,4 +100,4 @@ See full [documentation](https://github.com/antonmedv/fx/blob/master/docs.md).
 
 ## License
 
-[MIT](https://github.com/antonmedv/fx/blob/master/LICENSE)  
+The [`fx`](https://www.npmjs.com/package/fx) package is is open-sourced software licensed under the [MIT license](LICENSE).
```

Comment:
Nice sentence. But why you think it's better than mine? 😃

Reply:
I think a message is better than just a word. :)

Anyway, I will close this pull request. But you should add a little information about the contributing. It should be added, shouldn't it? :)

Reply:
There is no much contributing going on in fx. I think this is unnessesary. And I'll explain new contributors by myself, it one will come.

Reply:
Well, I just want to know what I can do or what I have to follow at this repository. Thank you for creating this pretty package! I am looking forward to seeing new wonderful things.